types/win-ca/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for win-ca 3.5
+// Project: https://github.com/ukoloff/win-ca
+// Definitions by: Marcos Junior <https://github.com/junalmeida>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { pki } from "node-forge";
+
+declare namespace Api {
+    enum der2 {
```

Comment:
where is the source for this?
Does it require 'enum' (considered not native JS object):
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums
(opening statement about modern TS)